displayio_cartesian.py

```diff
@@ -114,7 +116,7 @@ class Cartesian(Widget):
 
     **Summary: Cartesian Features and input variables**
 
-    The `Cartesian` widget has some options for controlling its position, visible appearance,
+    The `cartesian` widget has some options for controlling its position, visible appearance,
```

Comment:
I re-ran the  actions after merging that display text PR, the next issue it reports is from sphinx from this line.

This one needs to be capital C for `Cartesian` I think. It's building docs successfully after that chagne.

Reply:
Good catch, thanks!